main.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"flag"
+	"io"
+	"log"
+	"math/rand"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/go-chi/chi/v5/middleware"
+	"github.com/goccy/go-json"
+	"github.com/klauspost/compress/gzip"
+)
+
+const (
+	serverTimeout = time.Minute
+)
+
+var config struct {
+	serverPort         string
+	failAuthChance     float64
+	slowReceiveChance  float64
+	badResponseChance  float64
+	randomNetworkLag   int
+	disableJsonParsing bool
+}
+
+func main() {
+	parseConfig()
+
+	log.Println("starting the server at port " + config.serverPort)
```

Comment:
why not use the logger and config / flags-by-reflection from gotils?

Reply:
stdlib stuff is simpler and achieves the same result without any additional dependencies, don't see much value in gotils